pkg/controller/networkpolicy/store/networkpolicy.go

```diff
@@ -113,6 +113,10 @@ func ToNetworkPolicyMsg(in *types.NetworkPolicy, out *networking.NetworkPolicy,
 	out.Rules = in.Rules
 	out.AppliedToGroups = in.AppliedToGroups
 	out.Priority = in.Priority
+	if in.TierPriority != nil {
+		inTierPriority := uint32(*in.TierPriority)
+		out.TierPriority = &inTierPriority
+	}
```

Comment:
I think we assume objects in store will always be overridden when it's updated, so it should be safe to just copy the pointer like other fields. `AppliedToGroups` is a slice, `Priority` is `*float64`, we don't make a copy too.

Reply:
updated.. needed to do conversion from type TierPriority to uint32 .. moved the TierPriority type to networking types.. so this conversion is no longer needed. PTAL